admin/workspace-management/self-contained-builds.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
@Emyrk Could you please review this doc and let me know if there's anything incorrect/missing?

Reply:
Cool it gets its own page! In the future, there is a good chance this will become the default way we do things. But considering this is in Alpha, that will be a few releases out.